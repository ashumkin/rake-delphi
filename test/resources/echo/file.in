${echo_variable} must be replaced by its <value>