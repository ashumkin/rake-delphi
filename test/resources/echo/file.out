echo_variable_value must be replaced by its <value>
